google-cloud-bigquerystorage/src/test/proto/jsonTest.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto2";
+
+package com.google.cloud.bigquery.storage.test;
+
+message NestingLvl1 {
+  optional int64 test_int = 1;
+  optional NestingLvl2 nesting_value = 2;
+}
+
+message NestingLvl2 {
+  optional int64 test_int = 1;
+}
+
+message NestingStackedLvl0 {
+  optional int64 test_int = 1;
+  repeated NestingLvl1 nesting_value1 = 2;
+  optional NestingLvl2 nesting_value = 3;
+}
+
+message OptionTest {
+  optional int64 test_optional = 1;
```

Comment:
I thought the name is "scope" + filed name? Should this be root_test_optional?

Reply:
So the "root" I used there is actually just to prevent naming issues when building the protocol buffer. The actual name of the field is still going to be normal like test_int, nesting_value1, nesting_value, etc. I was debating if I should let the top most level be called root [for ex: root.test_optional] vs a single . [for ex: .test_optional], what do you think? The . vs root naming issue is going to be in the other PR though as it won't affect this one.